backend/src/services/emails_assets/reset_password.html

```diff
@@ -0,0 +1,81 @@
+  </div>
+  <div role="article" aria-roledescription="email" aria-label lang="en">
+    <div class="sm-px-4" style="background-color: #f8fafc; font-family: Inter, ui-sans-serif, system-ui, -apple-system, 'Segoe UI', sans-serif">
+      <table align="center" style="margin: 0 auto" cellpadding="0" cellspacing="0" role="none">
+        <tr>
+          <td style="width: 552px; max-width: 100%">
+            <div role="separator" style="line-height: 24px">&zwj;</div>
+            <table style="width: 100%" cellpadding="0" cellspacing="0" role="none">
+              <tr>
+                <td class="sm-p-6" style="border-radius: 8px; background-color: #fffffe; padding: 24px 36px; border: 1px solid #e2e8f0">
+                  <a href="https://hack4krak.pl">
+                    <img src="https://media.discordapp.net/attachments/1148574122536865822/1336744109930709034/logo_hack4krak_black.png?ex=67a4eb77&is=67a399f7&hm=761f144002623b653543f5d49253e41454903167929764d835f2f0dea47ba358&=&format=webp&quality=lossless&width=454&height=454" width="85" alt="Hack4Krak" style="max-width: 100%; vertical-align: middle">
+                  </a>
+                  <div role="separator" style="line-height: 24px">&zwj;</div>
+                  <h1 style="margin: 0 0 24px; font-size: 24px; line-height: 32px; font-weight: 600; color: #0f172a">
+                    Cześć, %user%!
+                  </h1>
+                  <p style="margin: 0 0 24px; font-size: 16px; line-height: 24px; color: #475569">
+                    Z twojego konta zostało wysłane rządanie zresetowania hasła. Aby to zrobić skopiuj poniższy kod i otwórz link podany poniżej.
```

Comment:
I think we already have email form for generic emails, so just use it instead of coyung entire html